types.ts

```diff
@@ -0,0 +1,11 @@
+import { IIdentityConfig } from "./types.d";
```

Comment:
This seems to have broken all types

![image](https://github.com/user-attachments/assets/0fd78756-6b6a-4de2-bbca-beb356660a42)

Reply:
Moved the type guards to the `utils` module.

Reply:
Yeah I think that's a much better place, types.ts is often just types.